### DISCUSSION TITLE
What does the market checkbox do?

### DISCUSSION THREAD

**Discussion by da2x** [Tribler/tribler#5612] [Q&A] [answered]
Enabling the market option in settings doesn’t seem to do anything. What is it supposed to do?

**Comment by devos50** [comment, marked as answer]
We have integrated [a fully decentralized marketplace](https://github.com/tribler/anydex-core) in Tribler. Since this is an experimental feature, we have disabled this component by default, but you found the setting to enable it 👍 

When you enable the marketplace, you will join the decentralized overlay in which orders can be made and shared. You can access the market GUI when you click on the token balance (in the upper-right corner), and then by pressing the 'trade' button.

We envision that our decentralized market becomes a key component in our [trust ecosystem](https://github.com/Tribler/tribler/wiki/The-design-of-a-trustworthy-ecosystem-around-media-sharing) that rewards users for good behaviour.

A scientific article that describes the matchmaking and settlement protocols can be found [here](https://arxiv.org/pdf/2004.05046).
